.github/workflows/build_msi_installer.yml

```diff
@@ -108,6 +108,19 @@ jobs:
           setupType: promptflow_with_extra
           scriptPath: ${{ env.testWorkingDirectory }}
 
+      - name: Install wheel
+        shell: pwsh
+        run: |
+          Set-PSDebug -Trace 1
+          pip install -r ${{ github.workspace }}/src/promptflow/dev_requirements.txt
+          pip install ${{ github.workspace }}/src/promptflow-tracing
+          pip install ${{ github.workspace }}/src/promptflow-core
```

Comment:
please use promptflow-core[executor-service] here, Not sure why fastapi not installed in step: Setup and Install dev promptflow

Reply:
fastapi is a testing package?